checker/src/main/java/org/checkerframework/checker/nullness/qual/EnsuresMultiNonNullIf.java

```diff
@@ -0,0 +1,27 @@
+package org.checkerframework.checker.nullness.qual;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.ConditionalPostconditionAnnotation;
+import org.checkerframework.framework.qual.InheritedAnnotation;
+
+/**
+ * An annotation wrapper to allow multiple postcondition annotations.
+ *
+ * <p>Programmers generally do not need to use this; it is created by Java when a programmer writes
+ * more than one {@code @EnsuresNonNullIf} annotation at the same location.
+ *
+ * @see EnsuresNonNullIf
+ * @checker_framework.manual #java-expressions-as-arguments Syntax of Java expressions
+ */
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.CONSTRUCTOR})
+@ConditionalPostconditionAnnotation(qualifier = NonNull.class)
+@InheritedAnnotation
+public @interface EnsuresMultiNonNullIf {
+    EnsuresNonNullIf[] value();
```

Comment:
Please write a Javadoc comment on every annotation element (field).

Reply:
Is this Ok?